scene/scene_string_names.h

```diff
@@ -58,7 +58,6 @@ class SceneStringNames {
 	StringName dot;
 	StringName doubledot;
 	StringName draw;
-	StringName hide;
```

Comment:
You'd have to remove it in the .cpp too :)

Reply:
My bad. I'm too used to force pushing on my own branch before merging when what I'm doing is not so public. I've reverted this change and gone with your second suggestion.